ddtrace/tracer/tracer_test.go

```diff
@@ -1050,13 +1049,40 @@ func BenchmarkStartSpan(b *testing.B) {
 }
 
 // startTestTracer returns a Tracer with a DummyTransport
-func startTestTracer(opts ...StartOption) (*tracer, *dummyTransport, func()) {
-	transport := newDummyTransport()
-	o := append([]StartOption{withTransport(transport)}, opts...)
+func startTestTracer(t interface {
+	// support both *testing.T and *testing.B
+	Fatalf(format string, args ...interface{})
+}, opts ...StartOption) (trc *tracer, transport *dummyTransport, flush func(n int), stop func()) {
```

Comment:
We're starting to have a lot of return values, and the tests are seeing a lot of `_, _,`. Should we consider returning a struct with a `*dummyTransport` and `.Stop()` and `.Flush()` implemented? 
It's entirely aesthetic, but I think it might be nicer than adding another positional value.

Reply:
You're right, this is getting pretty gross with all the return values and the interface. I was thinking to possibly do this as part of another PR, where we would clean up the tests more thoroughly. That task is actually on our "Tech Debt" list for quite a while now...